.claude/templates/do-skill/INTEGRATION_GUIDE.md

```
@@ -0,0 +1,465 @@
+      "enabled": false
+    }
+  }
+}
+```
+
+## Testing Your Integration
+
+After integrating, test with a simple task:
+
+1. **Create a test task:**
+   ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)